### PR TITLE
mark outdated blog post as such

### PR DESCRIPTION
**Pull request by bmeck**
see https://twitter.com/izs/status/1481293331963707398

### CODE REVIEW

**Review by nschonni — APPROVED** [review]

**Review by tniessen — APPROVED** [review]

**Review by Trott — APPROVED** [review]
LGTM and if you want to even farther than this, here's my take: Given that this post also exists at https://www.davepacheco.net/blog/post/2012-02-27-managing-node-js-dependencies-with-shrinkwrap/, I wouldn't mind removing it entirely and replacing it with a note that indicates (in no particular order):

* The post exists at https://www.davepacheco.net/blog/post/2012-02-27-managing-node-js-dependencies-with-shrinkwrap/
* ...but it is out of date...
* ...and so you probably want to refer to the npm docs instead.

In general, I'm for reducing/eliminating the blog, knowledge base, and tutorial content on the websites.

File: locale/en/blog/npm/managing-node-js-dependencies-with-shrinkwrap.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ layout: blog-post.hbs
 Photo by Luc Viatour (flickr)
 </p>
 
+**This post is outdated:** Please refer to official npm documentation on [shrinkwrap](https://docs.npmjs.com/cli/v8/configuring-npm/npm-shrinkwrap-json) and/or [lockfiles](https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json) going forward.
```
Comment (Trott) [inline locale/en/blog/npm/managing-node-js-dependencies-with-shrinkwrap.md]: Optional suggestion:

```suggestion
**This post is outdated.** Please refer to official npm documentation on [shrinkwrap](https://docs.npmjs.com/cli/v8/configuring-npm/npm-shrinkwrap-json) and/or [lockfiles](https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json) instead.
```

Another totally optional suggestion: Maybe hide the post content in a `<details>` tag that says "Original content" or something like that so people have to take an extra step to see it.

**Comment by bmeck** [comment]
fixup: Apply changes from code review comments (testing edit via code spaces)

**Comment by bmeck** [comment]
@Trott applied

**Review by Trott — APPROVED** [review]

**Pull request merged**